constants/constants.tsx

```diff
@@ -446,22 +446,22 @@ export const STARS = {
   id: "bg-stars",
   tiers: [
     {
-      count: 4,
+      count: 6,
       image: Star1,
       opacity: 0.5,
```

Comment:
```suggestion
      opacity: 0.3,
```
minor nit but can we make the slower moving stars have lighter opacity than the ones in front? so perhaps just flip the opacity for 0.3 and 0.5 in the tiers here